backend/src/api/routes/storage/index.routes.ts

```
@@ -442,9 +442,45 @@ router.get(
+      const isActive = ACTIVE_TYPES.has(baseType);
+
+      res.setHeader('Content-Type', rawType);
       res.setHeader('Content-Length', file.length.toString());
+      res.setHeader('X-Content-Type-Options', 'nosniff');
+      if (isActive) {
+        // RFC 6266: provide a quoted ASCII fallback plus a UTF-8
+        // percent-encoded filename* so non-ASCII keys round-trip safely.
+        const lastSegment = (objectKey.split('/').pop() || 'download').slice(0, 255);
+        const asciiFallback = lastSegment
+          .replace(/[^\x20-\x7e]+/g, '_')
+          .replace(/["\\\r\n]/g, '_');
```

ESLint

[error] Replace `⏎··········.replace(/[^\x20-\x7e]+/g,·'_')⏎··········` with `.replace(/[^\x20-\x7e]+/g,·'_')`

(prettier/prettier)